androidApp/src/androidTest/java/com/mbta/tid/mbta_app/android/nearbyTransit/NearbyTransitTabViewModelTest.kt

```diff
@@ -21,21 +21,22 @@ class NearbyTransitTabViewModelTest {
         var popCalled = false
         var pushedRouteCalled = false
 
-        vm.setStopFilter(null, newFilter, { popCalled = true }, { pushedRouteCalled = true })
-        assertFalse(pushedRouteCalled)
+        vm.setStopFilter(null, "a", newFilter, { popCalled = true }, { pushedRouteCalled = true })
+        assertTrue(pushedRouteCalled)
```

Comment:
This test was wrong as originally written - the test name describes testing that push is called.